docs/recipes/offline-transactions.md

```diff
@@ -1,3 +1,42 @@
 ---
 title: Sending Offline Transactions
----
\ No newline at end of file
+---
+
+# Offline Transaction
+
+## Durable Nonce
+
+`RecentBlockhash` is an important value for a tx. Your tx will be rejected if you use an expired recent blockhash (aprox. 2 mins). You can use `durable nonce` to get a never expired recent blockhash. To trigger this mechanism, you tx must
```

Comment:
Could we give the exact value(150 slots) for how long a blockhash is valid?

Let's also try and use full `transaction` instead of `tx`.

Also should be `To trigger this mechanism, your transaction must`

Reply:
fixed at [e106a4bf](https://github.com/solana-dev-adv/solana-cookbook/pull/21/commits/e106a4bf6a238ae62a609e8915a2bc4b14d66eb6)